<launch>
    <!--  get images -->
    <node name="pubimage" pkg="avp_slam_plus" type="pubimage" output="screen">
    </node>

    <!-- load Parameters -->
    <rosparam file="$(find avp_slam_plus)/config/configFile.yaml" command="load" />
   
    <!-- generate point cloud from RGB image of multi cameras -->
    <node name="pointCloudFromRGB" pkg="avp_slam_plus" type="pointCloudFromRGB" output="screen">
    </node>

　　 <!--  localization and mapping -->
    <node name="mapping" pkg="avp_slam_plus" type="mapping" output="screen">
    </node>

    <!--  loop closure -->
    <node name="PGO" pkg="avp_slam_plus" type="PGO" output="screen">
    </node>

    <node name="object_map_node" pkg="avp_slam_plus" type="object_map_node" output="screen">
    </node>
    <param name="data_folder" value="/home/zjl/Desktop/parking/avp_data/" />
    <param name="save_results_to_txt" value="false" />

</launch>
